qa/pull-tester/rpc-tests.sh

```diff
@@ -5,14 +5,21 @@ CURDIR=$(cd $(dirname "$0"); pwd)
 # Get BUILDDIR and REAL_BITCOIND
 . "${CURDIR}/tests-config.sh"
 
-export BITCOINCLI=${BUILDDIR}/qa/pull-tester/run-bitcoin-cli
 export BITCOIND=${REAL_BITCOIND}
 
 if [ "x${EXEEXT}" = "x.exe" ]; then
-  echo "Win tests currently disabled"
-  exit 0
+    export BITCOINCLI=${REAL_BITCOINCLI}
+    export EXEEXT=${EXEEXT}
+    echo 'Windows tests are disabled by default due to issue with Travis build'
+    echo 'To enable Widows tests, edit rpc-tests.sh and comment out "exit"'
+    exit
+else
+    export BITCOINCLI=${BUILDDIR}/qa/pull-tester/run-bitcoin-cli
 fi
 
+export BITCOIND=${REAL_BITCOIND}
```

Comment:
no reason, just an oversight, i'll take it out.

On 16/08/2015 5:25 AM, MarcoFalke wrote:
>
> In qa/pull-tester/rpc-tests.sh 
> <https://github.com/bitcoin/bitcoin/pull/6548#discussion_r37145294>:
>
> >  fi
> >
> > +export BITCOIND=${REAL_BITCOIND}
>
> Is there a reason to duplicate this from line 9?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/bitcoin/bitcoin/pull/6548/files#r37145294>.
>